src/components/Graph.react.js

```diff
@@ -18,11 +19,14 @@ const filterEventData = (gd, eventData, event) => {
         const data = gd.data;
         for(let i=0; i < eventData.points.length; i++) {
             const fullPoint = eventData.points[i];
-            const pointData = filter(function(o) {
+            var pointData = filter(function(o) {
                 return !contains(type(o), ['Object', 'Array'])
             }, fullPoint);
-
-            if (has('customdata', data[pointData.curveNumber]) &&
+            if(data[0]['type'].toLowerCase() === 'pie'){
+              return omit(pointData, ['originalEvent', 'cxFinal', 'cyFinal', 'cy', 'cx', 'vTotal']);
+            } else if (data[0]['type'].toLowerCase() === 'sankey'){
+              return pointData
```

Comment:
So instead of returning an array of points, we're just returning the single object. Is that because you can never click or hover on multiple points in a `pie` or a `sankey`? That's true for `clicking` and `hovering`, but might not necessarily be true in the future for `selecting`.

I think that we should return everything in an array for consistency for now, even if that chart type doesn't have a user interaction that supports returning multiple points. 

If we were to change the structure to only display a single object instead of an array for chart types that don't support returning multiple points, then we should add this logic to the rest of those chart types (like `scatter3d` and all of the maps). I don't think that that is something that I want to maintain.